app/views/posts/show.html.erb

```diff
@@ -5,4 +5,45 @@
   <span>
     <%= @post.user.first_name %>
   </span>
-</article>
\ No newline at end of file
+
+</article>
+
+<% if @post.likes.find_by(user_id: current_user.id)%>
+  <%= button_to('dislike', post_like_path(@post, @post.likes.find_by(user_id: current_user.id).id), method: :delete)%>
+<% else %>
+  <%= form_with(model: [@post, @post.likes.build], local: true) do |f| %>
+    <%= f.submit 'like' %>
+  <% end %>
+<% end %>
+
+
+
+
+
+<%= pluralize(@post.likes.count, 'like') %>
+
+<hr>
+
+
+<h3>
+  Create a comment:
+</h3>
+<%= form_with(model: [@post, @post.comments.build], local: true) do |f| %>
+  <%= f.text_area :content, placeholder: 'Write something...' %>
+  <%= f.submit 'Comment' %>
+<% end %>
+
+
+<ul>
```

Comment:
- [ ] **[OPTIONAL]** We can also use partial for comments with collection rendering to DRY rending comments.